src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -384,6 +385,65 @@ class Cas2ApplicationTest : IntegrationTestBase() {
         }
       }
     }
+
+    @Test
+    fun `Get single submitted application returns 200 with timeline events`() {
+      `Given a CAS2 User` { userEntity, jwt ->
+        `Given an Offender` { offenderDetails, inmateDetails ->
+          cas2ApplicationJsonSchemaRepository.deleteAll()
+
+          val newestJsonSchema = cas2ApplicationJsonSchemaEntityFactory
+            .produceAndPersist {
+              withAddedAt(OffsetDateTime.parse("2022-09-21T12:45:00+01:00"))
+              withSchema(
```

Comment:
This seems like a lot of generic test set up that, do we need it? Could we move it off into some set up helper. If we did we'd hopefully make this test more readable and help the next test follow along.

Reply:
A ha! You did this in the very next commit, I'll leave this comment in as a woop!

Reply:
yah, there's a lot of boilerplate to get through and think it could be potentially abstracted more, but maybe next time I touch the code :)